lib/modules/dosomething/dosomething_reportback/dosomething_reportback.install

```diff
@@ -92,9 +92,13 @@ function dosomething_reportback_schema() {
       ),
     ),
     'primary key' => array('rbid'),
+    'unique keys' => array(
+      'rbid-uid-nid-run_nid' => array('rbid', 'uid', 'nid', 'run_nid'),
```

Comment:
I'm pretty sure you don't need to add `rbid` here since it's serial, and the primary key..it's already unique